common/src/main/java/earth/terrarium/hermes/api/defaults/TextContentTagElement.java

```diff
@@ -24,11 +24,20 @@ public class TextContentTagElement extends FillAndBorderElement implements TagEl
 
     protected MutableComponent component = Component.empty();
     protected Alignment align;
+    protected boolean shadowed;
 
     public TextContentTagElement(Map<String, String> parameters) {
         super(parameters);
-
+        this.component.setStyle(Style.EMPTY
+                .withBold(ElementParsingUtils.parseBoolean(parameters, "bold", false))
+                .withItalic(ElementParsingUtils.parseBoolean(parameters, "italic", false))
+                .withUnderlined(ElementParsingUtils.parseBoolean(parameters, "underline", false))
+                .withObfuscated(ElementParsingUtils.parseBoolean(parameters, "obfuscated", false))
+                .withStrikethrough(ElementParsingUtils.parseBoolean(parameters, "strikethrough", false))
+                .withColor(ElementParsingUtils.parseColor(parameters, "color", Color.DEFAULT).getValue())
+        );
         this.align = ElementParsingUtils.parseAlignment(parameters, "align", Alignment.MIN);
+        this.shadowed = ElementParsingUtils.parseBoolean(parameters, "shadowed", false);
```

Comment:
shadowed should be true by default

Reply:
Genuinely curious: why? `<component>` was never default shadowed (nor any other tag?).

Zooming out a bit... I was hoping `<text>` would be the replacement for `<p>`, from a user POV.

From a code POV I thought maybe `<text>` would be the replacement of `TextTagElement`, then having `<p>` and `<text>` both just deserialize to / be synonyms for that, keeping `HeadingTagElement` as a subclass, `<component>` as a thin subclass for backwards compatibility reasons.

Well, that was were I surmised you were heading. The naming confusion of the TexTag stuff would go away then (`TextTagElement`, `TextTagElements`, TextContentTagElement`, etc.)

(this is in draft PR #25, which was opened mainly just for the conversation)

Reply:
I think shadowed by default is to follow vanilla in this, as vanilla defaults to true, if others are not they should to be to follow this standard. I also agree text can replace all pure text elements I dont think for component, but component can likely be deprecated as text replaces it, I dont see a way to safely backwards compat with it.

Reply:
711a29c defaults `TextContentTagElement` to shadowed.